__tests__/app.test.js

```diff
@@ -254,3 +254,18 @@ describe("/api/comments/:comment_id", () => {
       });
   });
 });
+describe("/api/users", () => {
+  test("GET: 200, responds with an array of all users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.users.length).toBe(4);
```

Comment:
maybe just testing for users.length being greater than zero would be better if we're taking a wider view  - could be  possible additions to your test data in a real world setting.